src/Administration/Resources/app/administration/src/module/sw-cms/page/sw-cms-list/index.js

```diff
@@ -94,6 +95,10 @@ export default {
 
         listCriteria() {
             const criteria = new Criteria(this.page, this.limit);
+            criteria.addAssociation('categories');
+            criteria.getAssociation('categories').setLimit(this.associationLimit);
+            criteria.addAssociation('products');
+            criteria.getAssociation('products').setLimit(this.associationLimit);
```

Comment:
i mean you can just do `criteria.getAssociation('products').setLimit(this.associationLimit);`

when you call `getAssociation` (also in php), the association is automatically added to the criteria.

Reply:
Good to know, will change it